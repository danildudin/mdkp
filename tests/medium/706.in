28 10
939 301 211 294 696 996 342 153 172 810 281 2 641 31 968 864 176 399 986 768 709 225 572 185 794 693 539 163 
409 594 333 572 914 529 260 981 332 391 1 980 672 199 515 844 143 174 701 302 750 570 920 725 889 992 377 595 6138
213 322 959 205 559 835 827 873 732 531 796 989 722 344 839 679 981 170 705 871 404 800 450 996 423 1000 47 286 5325
278 51 665 697 337 653 950 348 683 374 600 630 839 498 388 205 486 573 319 379 395 678 159 471 399 664 284 998 5587
799 224 103 398 162 970 944 189 708 266 103 659 602 366 394 226 821 633 939 977 505 888 672 664 688 277 207 439 5466
40 188 722 307 951 950 260 290 704 289 43 776 861 914 357 647 500 347 806 313 709 962 236 435 319 43 481 321 9565
298 673 906 629 165 59 915 713 213 286 451 267 212 388 409 160 930 423 620 73 154 328 379 532 762 607 720 55 9955
377 652 220 375 572 458 879 47 320 453 666 279 67 446 986 167 335 385 52 250 323 227 984 389 935 76 16 405 5683
849 1000 655 846 839 455 817 943 997 330 506 233 655 456 182 330 814 819 570 675 729 178 188 456 323 10 228 677 8242
729 662 99 424 289 351 777 712 867 126 386 833 40 451 613 208 505 203 649 28 411 673 367 151 318 844 881 42 8447
78 66 650 887 679 289 801 313 169 630 73 319 973 658 769 983 155 127 741 876 870 24 357 631 25 188 763 43 8505
