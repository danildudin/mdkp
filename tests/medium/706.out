8073
10
0 4 9 14 15 18 19 20 24 26 
